qiskit/circuit/library/standard_gates/ecr.py

```diff
@@ -110,7 +110,15 @@ def _define(self):
         self.definition = qc
 
     def inverse(self, annotated: bool = False):
-        """Return inverse ECR gate (itself)."""
+        """Return inverse ECR gate (itself).
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
I might reword this here slightly. Maybe something like:

```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this gate is self inverse.
```

I feel like this would make it clearer on what the argument is normally used for but also critically in this case that it's ignored. I tried to write this in a way so you can reuse it for all the standard gates that ignore the annotated flag. The only difference is you'll have to change self inverse for things like s and sdg.